src/App.js

```diff
@@ -47,14 +48,19 @@ class App extends Component {
       dopplerSession: { status: sessionStatus, userData },
       i18nLocale,
     } = this.state;
+    const redirectToReports = () => <Redirect to={{ pathname: '/reports' }} />;
+
     return (
       <DopplerIntlProvider locale={i18nLocale}>
         {sessionStatus === 'authenticated' ? (
-          <>
-            <Header userData={userData} />
-            <Reports />
-            <Footer />
-          </>
+          <Router>
+            <div>
+              <Header userData={userData} />
+              <Route path="/" exact component={redirectToReports} />
+              <Route path="/reports/" exact component={Reports} />
```

Comment:
in doppler the routes are in Title Clase like `Reports` . no need maintain consistency?''''''''''''

Reply:
Mmm, in general apps has lowercase URLs, in my opinion, we should take advantage of this opportunity to change it.

Reply:
I did not get answer on this, I think that in general Doppler's URLs are really ugly, so I do not see bad take this opportunity.

Merging...